src/window.cpp

```diff
@@ -98,10 +101,10 @@ window_impl::window_impl(int pWidth, int pHeight, const char* pTitle,
         }
     }
 
-    // Set context (before glewInit())
+    /* Set context (before glewInit()) */
     MakeContextCurrent(this);
 
-    //GLEW Initialization - Must be done
+    /* GLEW Initialization - Must be done */
```

Comment:
May be you misunderstood, that is not what i meant. This can happen by accidentally deleting the new line at end of comment. I didn't mean some one would write a valid code statement and comment in same line.